Sources/SwiftDocC/Semantics/Options/Options.swift

```diff
@@ -0,0 +1,46 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+public class Options: Semantic, AutomaticDirectiveConvertible {
+    public let originalMarkup: BlockDirective
+    
+    @DirectiveArgumentWrapped
+    public private(set) var scope: Scope = .local
+    
+    @ChildDirective
+    public private(set) var automaticSeeAlso: AutomaticSeeAlso? = nil
```

Comment:
These are the names of directives, but some of these can be confusing as property types when accessed from other pieces of the codebase. For example, I'd expect this property to be a See Also section type rather than a type that describes the behavior of See Also sections. Maybe these should be private and `Options` could get more semantic computed properties, like `automaticSeeAlsoBehavior`

Reply:
Good point. I've replaced these with underscored properties and property wrappers that expose the behaviors directly.